### ISSUE TITLE
There is no make docker-agent in makefile as said in README under demos

### ISSUE THREAD

**Issue by etsangsplk** [open-telemetry/opentelemetry-service#24]
README under https://github.com/open-telemetry/opentelemetry-service/tree/master/demos/trace
writes about 
```
make docker-agent && make docker-collector```

There is no docker-agent  in make file, only docker-collector

**Comment by songy23** [comment]
Those are just removed in https://github.com/open-telemetry/opentelemetry-service/pull/21.

**Comment by flands** [comment]
It is highly recommended you use the OC Service repo for now -- with that said, will put up a bandaid PR to address.

**Comment by songy23** [comment]
Fixed in #32.
